studio/components/interfaces/Organization/NewProject/NotOrganizationOwnerWarning.tsx

```diff
@@ -8,12 +8,11 @@ const NotOrganizationOwnerWarning = () => {
         icon={<IconAlertCircle className="text-white" size="large" strokeWidth={1.5} />}
         defaultVisibility={true}
         hideCollapse
-        title="You do not have permission to create a project"
+        title="You do not have permissions to create a project"
```

Comment:
nit: I think the old one is maybe more correct?
Or maybe: You do not have _the correct_ permissions to create a project

Reply:
yeap! thats fair, will go with the former